api/authenticators/saml.go

```diff
@@ -71,15 +72,25 @@ func (self *SamlAuthenticator) AddHandlers(mux *http.ServeMux) error {
 		return err
 	}
 
-	samlMiddleware, err = samlsp.New(samlsp.Options{
+	opts := samlsp.Options{
 		IDPMetadata: idpMetadata,
 		URL:         *rootURL,
 		Key:         key,
 		Certificate: cert,
-	})
+	}
+	samlMiddleware, err = samlsp.New(opts)
 	if err != nil {
 		return err
 	}
+
+	maxAge := time.Minute * time.Duration(self.authenticator.DefaultSessionExpiryMin)
```

Comment:
This will be 0 if not defined you should get it the same way as https://github.com/Velocidex/velociraptor/blob/ecb35eb92667499dd59b7d3586df7e6b4d1d4f88/api/authenticators/common.go#L41

Reply:
Thanks, fixed now!